datarepo-clienttests/src/main/java/scripts/testscripts/CreateSnapshot.java

```diff
@@ -67,15 +96,32 @@ public void setup(List<TestUserSpecification> testUsers) throws Exception {
         loadSummary.getTotalFiles(),
         equalTo(loadSummary.getSucceededFiles()));
 
-    // generate load for the simple dataset
-    String testConfigGetIngestbucket = "jade-testdata";
-    String fileRefName =
-        "scratch/buildSnapshotWithFiles/" + FileUtils.randomizeName("input") + ".json";
-    BlobId scratchFile =
-        BulkLoadUtils.writeScratchFileForIngestRequest(
-            server.testRunnerServiceAccount, result, testConfigGetIngestbucket, fileRefName);
-    IngestRequestModel ingestRequest = BulkLoadUtils.makeIngestRequestFromScratchFile(scratchFile);
-    scratchFiles.add(scratchFile); // make sure the scratch file gets cleaned up later
+    if (cloudPlatform.equals(CloudPlatform.GCP)) {
+      // generate load for the simple dataset
+      String testConfigGetIngestbucket = "jade-testdata";
+      String fileRefName =
+          "scratch/buildSnapshotWithFiles/" + FileUtils.randomizeName("input") + ".json";
+      BlobId scratchFile =
+          BulkLoadUtils.writeScratchFileForIngestRequest(
+              server.testRunnerServiceAccount, result, testConfigGetIngestbucket, fileRefName);
+      ingestRequest = BulkLoadUtils.makeIngestRequestFromScratchFile(scratchFile);
+      scratchFiles.add(scratchFile); // make sure the scratch file gets cleaned up later
+    } else if (cloudPlatform.equals(CloudPlatform.AZURE)) {
+      String flightId = UUID.randomUUID().toString();
```

Comment:
Does this have to be called flightid?  It seems odd that this would be related to a flight

Reply:
Oh good point! I copied this from a different test